### PR TITLE
Map GameTest methods

### PR DESCRIPTION
**Pull request by apple502j**
Mapped GameTest methods. Not so confident with the names, but think it's sort of good.

### CODE REVIEW

File: mappings/net/minecraft/test/GameTest.mapping [hunk 1]
```diff
@@ -48,3 +55,16 @@ CLASS net/minecraft/class_4517 net/minecraft/test/GameTest
...
+	METHOD method_33315 tickTests ()V
+	METHOD method_33316 (Lnet/minecraft/class_4518;)V
+		ARG 1 listener
+	METHOD method_36059 getStructureBlockBlockEntity ()Lnet/minecraft/class_2633;
+	METHOD method_36060 runAtTick (JLjava/lang/Runnable;)V
+		ARG 1 tick
+		ARG 3 runnable
+	METHOD method_36061 getSize ()Lnet/minecraft/class_2382;
+	METHOD method_36062 getBoundingBox ()Lnet/minecraft/class_238;
+	METHOD method_36063 getElapsedMilliseconds ()J
```
Comment (liach) [inline mappings/net/minecraft/test/GameTest.mapping:67]: Just fyi, we usually call milliseconds like "time" and ticks "ticks", so you can use `getElapsedTime` if you feel `getElapsedMilliseconds` is too long. No need to change though

File: mappings/net/minecraft/test/XMLReportingTestCompletionListener.mapping [hunk 2]
```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/class_6307 net/minecraft/test/XMLReportingTestCompletionListener
```
Comment (liach) [inline mappings/net/minecraft/test/XMLReportingTestCompletionListener.mapping]: Use `XmlReporting...` than `XMLReporting`.

File: mappings/net/minecraft/test/TimedTaskRunner.mapping [hunk 3]
```diff
@@ -2,9 +2,24 @@ CLASS net/minecraft/class_4693 net/minecraft/test/TimedTaskRunner
...
 	METHOD method_23644 runReported (J)V
 		ARG 1 tick
 	METHOD method_23645 runTasks (J)V
 		ARG 1 tick
+	METHOD method_36077 (ILjava/lang/Runnable;)Lnet/minecraft/class_4693;
+		ARG 2 task
+	METHOD method_36078 createAndAdd (JLjava/lang/Runnable;)Lnet/minecraft/class_4693;
+		ARG 1 duration
+		ARG 3 task
+	METHOD method_36079 createAndAddDurationless (Ljava/lang/Runnable;)Lnet/minecraft/class_4693;
```
Comment (Juuxel) [inline mappings/net/minecraft/test/TimedTaskRunner.mapping]: Does this need `Durationless`? It looks like the duration variant just takes it as an extra parameter so they shouldn't conflict.

File: mappings/net/minecraft/test/TimedTask.mapping [hunk 4]
```diff
@@ -1,3 +1,11 @@
...
 	FIELD field_21450 duration Ljava/lang/Long;
 	FIELD field_21451 task Ljava/lang/Runnable;
+	METHOD <init> (Ljava/lang/Long;Ljava/lang/Runnable;)V
+		ARG 1 duration
+		ARG 2 task
+	METHOD method_35941 create (JLjava/lang/Runnable;)Lnet/minecraft/class_4692;
+		ARG 0 duration
+		ARG 2 task
+	METHOD method_35942 createDurationless (Ljava/lang/Runnable;)Lnet/minecraft/class_4692;
+		ARG 0 task
```
Comment (Juuxel) [inline mappings/net/minecraft/test/TimedTask.mapping:11]: Same here about `Durationless`

File: mappings/net/minecraft/test/TestServer.mapping [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+		ARG 1 serverThread
+		ARG 2 session
+		ARG 3 dataPackManager
+		ARG 4 serverResourceManager
+		ARG 5 batches
+		ARG 6 pos
+		ARG 7 registryManager
+	METHOD <init> (Ljava/lang/Thread;Lnet/minecraft/class_32$class_5143;Lnet/minecraft/class_3283;Lnet/minecraft/class_5350;Ljava/util/Collection;Lnet/minecraft/class_2338;Lnet/minecraft/class_5455$class_5457;Lnet/minecraft/class_2378;Lnet/minecraft/class_2378;)V
+		ARG 1 serverThread
+		ARG 2 ession
```
Comment (Juuxel) [inline mappings/net/minecraft/test/TestServer.mapping]:
```suggestion
		ARG 2 session
```

File: mappings/net/minecraft/test/TestFunctions.mapping [hunk 6]
```diff
@@ -24,3 +28,16 @@ CLASS net/minecraft/class_4519 net/minecraft/test/TestFunctions
 	METHOD method_29406 clearFailedTestFunctions ()V
 	METHOD method_32244 (Ljava/lang/String;)Ljava/util/function/Consumer;
 		ARG 0 batchId
+	METHOD method_36069 (Ljava/lang/reflect/Method;)V
+		ARG 0 method
+	METHOD method_36070 validateCount (Ljava/lang/reflect/Method;Ljava/lang/Class;Ljava/util/function/Function;Ljava/util/Map;)V
+		ARG 0 method
+		ARG 1 class_
```
Comment (Juuxel) [inline mappings/net/minecraft/test/TestFunctions.mapping]:
```suggestion
		ARG 1 clazz
```

**Review by Juuxel — CHANGES_REQUESTED** [review]

**Review by liach — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Pull request merged**